features/step_definitions/authorization.js

```diff
@@ -37,4 +46,82 @@ module.exports = function server() {
             return null;
         });
     });
+
+    this.Then(/^they can see the project$/, () =>
+        request({                           // make sure pipeline exists (TODO: move to Given an existing pipeline with that repository scenario)
+            uri: `${this.instance}/${this.namespace}/pipelines`,
+            method: 'POST',
+            auth: {
+                bearer: this.jwt
+            },
+            body: {
+                scmUrl: `git@github.com:${this.repoOrg}/${this.repoName}.git#master`
+            },
+            json: true
+        }).then((response) => {
+            if (!this.pipelineId) {
+                this.pipelineId = response.body.id;
+            }
+
+            Assert.oneOf(response.statusCode, [409, 201]);
+        })
+        .then(() => request({
+            method: 'GET',
+            url: `${this.instance}/${this.namespace}/pipelines/${this.pipelineId}`,
+            followAllRedirects: true,
+            json: true
+        }))
+        .then((response) => {
+            Assert.strictEqual(response.statusCode, 200);
+        })
+    );
+
+    this.Then(/^they can start the "main" job$/, { timeout: 60 * 1000 }, () =>
+        request({
+            uri: `${this.instance}/${this.namespace}/pipelines/${this.pipelineId}/jobs`,
+            method: 'GET',
+            json: true
+        })
+        .then((response) => {
+            this.jobId = response.body[0].id;
+
+            Assert.equal(response.statusCode, 200);
+        })
+        .then(() =>
+            request({
+                uri: `${this.instance}/${this.namespace}/builds`,
+                method: 'POST',
+                body: {
+                    jobId: this.jobId
+                },
+                auth: {
+                    bearer: this.jwt
+                },
+                json: true
+            }).then((resp) => {
+                this.buildId = resp.body.id;
+
+                Assert.equal(resp.statusCode, 201);
+            })
+        )
+    );
+
+    this.Then(/^they can delete the pipeline$/, { timeout: 60 * 1000 }, () =>
+        this.waitForBuild(this.buildId).then(response => {
```

Comment:
I was worried things wouldn't be cleaned up properly we tried to delete the pipeline when the build was still building.
Will remove since it doesn't seem to be an issue.